src/WebSocketServer/WebSocketServerPlugin.cs

```diff
@@ -85,7 +85,8 @@ public override void Dispose()
         {
             _host?.Dispose();
             _connections?.Dispose();
-            _neoSystem.MemPool.TransactionAdded -= OnMemPoolTransactionAdded;
+            if (_neoSystem != null)
```

Comment:
should not you just  stop processing if _neoSystem is null

Reply:
No, if `neosystem` is not null then we need to remove the handler, or else it will crash on a new item sent to the `mempool`.

Reply:
Ur right, I did not notice its dispose.